app/lib/shared/versions.dart

```diff
@@ -39,10 +39,10 @@ final String runtimeSdkVersion = '2.7.0';
 final String toolEnvSdkVersion = '2.7.0';
 
 // keep in-sync with app/pubspec.yaml
-final String panaVersion = '0.13.1+4';
+final String panaVersion = '0.13.2';
```

Comment:
I think pana knows its own version. Could we import this from pana?

Reply:
Hm, sure, we could do that, I'll do a follow-up.